_posts/2019-10-16-monthly-dev-post.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Monthly Dev Post October 2019
+author: alberth
+---
+
+Welcome to our Dev Post for October 2019.
+Every month (ahem) one of these post will be created, to tell you about the latest developments in the world of OpenTTD.
+
+<!-- more -->
+
+## Development
+
+### 1.9.3 Released
+
+OpenTTD [1.9.3 update](https://www.openttd.org/news/2019/09/16/openttd-1-9-3.html) was released in September.
+
+This wrapped up a bunch of bug fixes and and performance improvements, thanks to all contributors.
```

Comment:
Perhaps end with an exclamation point, since the sentence structure implies you are exclaiming your thanks to the contributors.